packages/venia-ui/package.json

```diff
@@ -47,6 +47,7 @@
     "react-slick": "~0.25.2",
     "redux-actions": "~2.6.4",
     "redux-thunk": "~2.3.0",
+    "tabbable": "~5.1.2",
```

Comment:
We're willing to accept the size of this lib if you can guarantee it doesn't end up in the main vendor bundle; this might require that `FilterModal` is lazy loaded if it already didn't shake out that way on its own. You can run `yarn stats:venia` to compare bundle sizes before and after this change.

Reply:
https://github.com/magento/pwa-studio/pull/2843 @tjwiebell this pr introduces `react-aria` for focus trap, if that is merged I will rewrite this pr.

Reply:
Good eye; this issue did get a little outdated and we very recently started experimenting with the Adobe maintained `react-aria` library, which we probably should have included as implementation guidance. Since this solution with the tabbable lib worked, and we didn't provide that guidance, we were prepared to accept as-is, and re-factor later. I think we're confident enough in the results of that Spike that `react-aria` would be our preferred solution, using the `FocusScope` HOC here.

I don't know the timeline of that Spike getting delivered, but if you would like to preemptively switch this implementation to `react-aria`, it would be very much appreciated; but again, very much optional.